custom-recipes/nlp-amazon-translation-translate/recipe.py

```diff
@@ -7,11 +7,11 @@
 from dataiku.customrecipe import get_recipe_config
 from dataiku.customrecipe import get_input_names_for_role
 from dataiku.customrecipe import get_output_names_for_role
+from dkulib.dkulib.dku_io_utils import set_column_descriptions
```

Comment:
Any way we can do
```suggestion
from dkulib.dku_io_utils import set_column_descriptions
```
Perhaps a change we should do to dkulib?

Reply:
Yeah we can add the imports to the __init__.py in dkulib -- Let me open a PR for that

Reply:
Although then it'll be
`from dkulib import set_column_descriptions`
That good as well @alexcombessie ?

Reply:
I think that simpler is better, so yes. Does it prevent us from introducing submodules with dkulib?

Reply:
I don't think so

Reply:
perfect, good to merge on both sides

Reply:
Hmm now that the imports are added to the `__init__.py` in dkulib, it will force us to also have all requirements of all submodules, since there is e.g. import cld3 [here](https://github.com/dataiku/dss-plugin-dkulib/blob/fc408033f9531a7823e466c9246ae874e85c1004/dkulib/nlp/language_detector.py#L9) 

I see two options:
a) Selectively include imports in `__init__.py` in dkulib, e.g. only having the parallelizer there will only force us to have its requirements
b) Backtrack and just leave it as `from dkulib.dkulib.dku_io_utils import set_column_descriptions`

I'm in clear favor of b) as with a) this will again force us to edit the submodules
What do you think and is there an option c)? @mhham @alexcombessie 
I should have thought of this in advance😅

Reply:
I would vote c) Move the submodules parallelizer, dku_config etc. out of the dkulib folder. Effectively, this module don't have much to do with one another. Several of them are not even dku specific. 
This way, you avoid forcing to have an `__init__.py` which imports everything. You just leave have one init per module, which import what is needed.

Reply:
This is actually a very good exercise which forces us to think of dkulib as an actual library used by other projects!

Reply:
I change my vote to c) as well haha! I'll open a PR for this in dkulib

Reply:
Thanks Niklas !

Reply:
Since we import everything in the root `__init__.py`, choosing b) without modifying the dss-dkulib library will not suffice.

We need to have an empty `__init__.py` at the root instead of importing everything to make it work without importing everything

Reply:
Yeah sorry that's what I meant, i.e. revert the PR in dkulib & leave the imports as is here -- Are you okay with c) @mhham ?

Reply:
Based on the discussion [here](https://github.com/dataiku/dss-plugin-dkulib/pull/22#issuecomment-887562713), we decided to leave the imports as is, i.e. `from dkulib.dkulib.dku_io_utils import set_column_descriptions`